packages/core/nuxt-theme-module/theme/layouts/error.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="require('@storefront-ui/shared/icons/error.svg')"
+    />
+    <SfHeading
```

Comment:
https://nuxtjs.org/docs/2.x/concepts/views#error-page

We should do accoridng to this doc. The error page will be shown for all errors so its best to catch 404 with proper message and display "An error occured" for other ones with a status code as a title ;)